.github/workflows/CreateMilestone.yml

```diff
@@ -12,3 +12,4 @@ jobs:
       - uses: sonarsource/gh-action-lt-backlog/CreateRspecIssue@v1
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
+          column-id: 4971951
```

Comment:
Could you please extract a constant? It would be great to have a name for this id.

Reply:
Added comment